### DISCUSSION TITLE
HTTP Error 500.30 Error

### DISCUSSION THREAD

**Discussion by FierceMuppet** [quartznet/quartznet#1098] [Q&A] [answered]
I have a .NET 3.1 Core project that was working fine with Quartz.  All of a sudden, I am getting HTTP Error 500.30 error.  The occurs as soon as I add the job. in CreateHostBuilder.  If I comment out the add job, it works fine.

I tried just stripping it down to the hello world ijob and have the same result.

Any ideas?

**Comment by lahma** [comment, marked as answer]
If you're on Azure, [this guide might help](https://zimmergren.net/solving-asp-net-core-3-on-azure-app-service-causing-500-30-in-process-startup-failure/) . Sometimes something as simple as restart might help if it's transient error. Without logs it's quite impossible say.
